javascript/ecmascript-2015/number-extensions/number-extensions-part-1.md

```diff
@@ -93,23 +104,15 @@ Complete the missing calls with methods such that all the statements are true:
 
 ```javascript
 
-Number.???(5) // true
-???.???(2.71) // false
-Number.???(Number.NaN) // true
+Number.isNaN('enki')
+Number.isFinite(0)
+Number.isInteger(9.81)
+// ???
 ```
 
-* isFinite
-* Number
-* isInteger
-* isNaN
-* isReal
-* String
-* isRatio
-* NaN
-* isNotNaN
- 
----
-## Footnotes
+* false, true, false
+* false, false, false
+* true, true, false
+* false, true, true
+* true, false, true
```

Comment:
these answers might be a bit hard to choose, as you have to do a bit of searching between the answers to find the right one:

![image](https://user-images.githubusercontent.com/19802557/82330059-3a677380-99da-11ea-867b-ed2c551d4af5.png)

we could do a mix of the old question and the new one, something like:
``````
```javascript
Number.isNan('enki');
// ???
Number.isFinite(0);
// ???
Number.???(9.81);
// false
```

* false
* true
* isInteger
* isNaN
``````

what do you think @nem035 ?

Reply:
Ideally, we'll use value comparison here but that won't be ready for a bit longer.

I'm ok with changing it for now if you'd like, no strong preference either way

Reply:
hmmm, let's leave it as is for now and improve when we have value comparison :+1: